ada/extensions/src/libadalang-helpers.ads

```diff
@@ -34,6 +34,11 @@ package Libadalang.Helpers is
    package Unit_Vectors is new
      Ada.Containers.Vectors (Positive, Analysis_Unit);
 
+   Abort_App_Exception : exception;
+   procedure Abort_App (Message : String := "") with No_Return;
```

Comment:
This design is a bit strange: I would have expected either:

1. just a procedure Abort_App that takes a message (and a private hidden exception as an implementation detail).
2. just an exception, which when raised will be caught by the app framework, and print the embedded message on stderr before exiting.

Having both is a bit confusing for no reason IMHO. There's not much use in using the exception directly if you have the procedure.

Reply:
Fair point. I’ll go for the procedure-only way (easier to use, and no restriction on Message). Thanks!